<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="Rawna" name="TS_01.UpcomingRequirement" tests="7" failures="1" timestamp="26 Jun 2020 06:00:14 GMT" time="49.847" errors="0">
  <testcase name="startBrowser" time="0.429" classname="TS_01.UpcomingRequirement"/>
  <testcase name="loadUpcomingBikesPage" time="1.367" classname="TS_01.UpcomingRequirement"/>
  <testcase name="navigateToUpcomingBikesPage" time="0.031" classname="TS_01.UpcomingRequirement">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at TS_01.UpcomingRequirement.navigateToUpcomingBikesPage(UpcomingRequirement.java:91)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- navigateToUpcomingBikesPage -->
  <testcase name="navigateToUpcomingBikesPage" time="0.094" classname="TS_01.UpcomingRequirement">
    <skipped/>
  </testcase> <!-- navigateToUpcomingBikesPage -->
  <testcase name="navigateToUpcomingBikesPage" time="0.183" classname="TS_01.UpcomingRequirement">
    <skipped/>
  </testcase> <!-- navigateToUpcomingBikesPage -->
  <testcase name="navigateToUpcomingBikesPage" time="0.573" classname="TS_01.UpcomingRequirement">
    <skipped/>
  </testcase> <!-- navigateToUpcomingBikesPage -->
  <testcase name="outputDisplay" time="47.170" classname="TS_01.UpcomingRequirement"/>
</testsuite> <!-- TS_01.UpcomingRequirement -->
